contrib/win32/win32compat/Debug.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-void     fatal(const char *, ...);
+void     __declspec(noreturn) fatal(const char *, ...);
```

Comment:
why is the special treatment for fatal() but not for error/verbose/debug?

Reply:
IntelliSense doesn't show these warnings.. which static analysis tool, you used?

Reply:
The fatal() function is the only call that will force the program to exit.  The others will not. The declaration informs the compiler the function will never return and will use this information to optimize compilation as well as assist in static code analysis (e.g., detecting unreachable code, etc).

In Visual Studio, you can do Analyze -> Run Code Analysis -> Run Code Analysis on <project name>.  However, the results aren't always perfect, particularly when it comes to checking bounds on dynamic memory allocations that are based on runtime-calculated values.